runners/reference/job-server/build.gradle

```diff
@@ -22,13 +22,17 @@ applyJavaNature()
 description = "Apache Beam :: Runners :: Reference :: Job Orchestrator"
 
 dependencies {
+  compile project(path: ":beam-runners-parent:beam-java-fn-execution", configuration: "shadow")
+  compile project(path: ":beam-runners-parent:beam-local-artifact-service-java", configuration: "shadow")
   shadow library.java.grpc_netty
   shadow project(path: ":beam-model-parent:beam-model-job-management", configuration: "shadow")
   shadow library.java.grpc_core
   shadow library.java.grpc_stub
   shadow library.java.slf4j_api
   shadow library.java.args4j
   testCompile library.java.junit
+  testCompile project(path: ":beam-runners-parent:beam-runners-core-construction-java")
```

Comment:
you want the shadow configuration of the project otherwise you'll get the unshaded jar

Reply:
Done, I think; `testShadow` in the `shadow` configuration, or still just `testCompile`

Reply:
testShadow is the most correct but this module doesn't shade the test jar so in reality there is no difference between testCompile and testShadow so both of those would work.